include/seqan3/std/concepts

```diff
@@ -435,167 +435,167 @@ SEQAN3_CONCEPT EqualityComparableWith =
         /* not required to be equality preserving */
     };
 //!\endcond
 
-/*!\interface   std::RegularInvocable <>
- * \extends     std::Invocable
+/*!\interface   std::regular_invocable <>
+ * \extends     std::invocable
  * \brief       Specifies whether the given callable is invocable with the given arguments and equality preserving
  *              (invocations change neither the callable, nor the arguments).
- * \sa          http://en.cppreference.com/w/cpp/concepts/RegularInvocable
+ * \sa          http://en.cppreference.com/w/cpp/concepts/regular_invocable
  */
 //!\cond
 template< class F, class... Args >
-SEQAN3_CONCEPT RegularInvocable = Invocable<F, Args...>;
+SEQAN3_CONCEPT regular_invocable = invocable<F, Args...>;
 //!\endcond
 
-/*!\interface   std::Predicate <>
- * \extends     std::RegularInvocable
- * \brief       Specifies whether the given callable is std::RegularInvocable and returns bool.
- * \sa          http://en.cppreference.com/w/cpp/concepts/Predicate
+/*!\interface   std::predicate <>
+ * \extends     std::regular_invocable
+ * \brief       Specifies whether the given callable is std::regular_invocable and returns bool.
+ * \sa          http://en.cppreference.com/w/cpp/concepts/predicate
  */
 //!\cond
 template < class F, class... Args >
-SEQAN3_CONCEPT Predicate = RegularInvocable<F, Args...> && Boolean<std::invoke_result_t<F, Args...>>;
+SEQAN3_CONCEPT predicate = regular_invocable<F, Args...> && boolean<std::invoke_result_t<F, Args...>>;
 //!\endcond
 
-/*!\interface   std::Relation <>
- * \extends     std::Predicate
+/*!\interface   std::relation <>
+ * \extends     std::predicate
  * \brief       Specifies that R defines a binary relation over the set of expressions whose type and value category
  *              are those encoded by either t or u.
- * \sa          http://en.cppreference.com/w/cpp/concepts/Relation
+ * \sa          http://en.cppreference.com/w/cpp/concepts/relation
  */
 //!\cond
 template <class R, class T, class U>
-SEQAN3_CONCEPT Relation =
-    Predicate<R, T, T> &&
-    Predicate<R, U, U> &&
-    CommonReference<std::remove_reference_t<T> const &, std::remove_reference_t<U> const &> &&
-    Predicate<R,
+SEQAN3_CONCEPT relation =
+    predicate<R, T, T> &&
+    predicate<R, U, U> &&
+    common_reference_with<std::remove_reference_t<T> const &, std::remove_reference_t<U> const &> &&
+    predicate<R,
         ::ranges::common_reference_t<std::remove_reference_t<T> const &, std::remove_reference_t<U> const &>,
         ::ranges::common_reference_t<std::remove_reference_t<T> const &, std::remove_reference_t<U> const &>> &&
-    Predicate<R, T, U> &&
-    Predicate<R, U, T>;
+    predicate<R, T, U> &&
+    predicate<R, U, T>;
 //!\endcond
 
-/*!\interface   std::StrictWeakOrder <>
- * \extends     std::Relation
- * \brief       The concept StrictWeakOrder<R, T, U> specifies that the Relation R imposes a strict weak ordering
+/*!\interface   std::strict_weak_order <>
```

Comment:
Why does strict stay in this concept name, and not in StrictTotallyOrdered and others?

Reply:
Ask the ISO committee...